resources/user-agents/browsers/chrome/chrome-66-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Chrome #MAJORVER#.#MINORVER#",
-  "versions": [85, 84, 83, 82, 81, 80, 79, 78, 77, 76, 75, 74, 73, 72, 71, 70, 69, 68, 67, 66],
+  "versions": [92, 91, 90, 89, 88, 87, 86, 85, 84, 83, 82, 81, 80, 79, 78, 77, 76, 75, 74, 73, 72, 71, 70, 69, 68, 67, 66],
```

Comment:
Please add these versions also to the Chromium and to the Android Version of Chrome

Reply:
Done!

Reply:
Hmm will need to add Chromium i guess, one minute

Reply:
Done!